roles/vivumlab_service-edit/tasks/main.yml

```
@@ -0,0 +1,17 @@
+---
+- name: Ensures override dir exists
+  file:
+    path: "settings/{{config_dir}}/override/{{ services[0] }}"
+    state: directory
+    owner: '1000'
+    group: '1000'
+
+- name: Create docker-compose.override.yml
+  copy:
```

If we template them, we need a template here instead of copy.
```suggestion
  template:
```